jina/serve/runtimes/worker/http_csp_app.py

```diff
@@ -74,7 +75,7 @@ def add_post_route(
             input_model,
             output_model,
             input_doc_list_model=None,
-            output_doc_list_model=None,
+            parameters_model=None,
```

Comment:
reviewing on phone, so cannot dig much deeper. Why is this removed?

Reply:
oh this `output_doc_list_model ` was never used